share/zm_testagent

```diff
@@ -0,0 +1,37 @@
+#!/bin/sh
+
+# PROVIDE: zm_testagent
+# REQUIRE: mysql postgresql
```

Comment:
What happens if not both of the database engines are installed?

Reply:
Ditto.